qemu/tests/cfg/slof_balloon.cfg

```diff
@@ -0,0 +1,19 @@
+- slof_balloon: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = slof_balloon
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    start_vm = no
+    pci_controllers = ''
+    balloon = balloon0
+    balloon_dev_devid = balloon0
+    balloon_dev_add_bus = yes
+    only ppc64le ppc64
+    variants:
+        - with_pci_bridge:
+            pci_controllers +=" pci_bridge0"
```

Comment:
I think Line 8 can be deleted and Line 15 can be changed to 'pci_controllers = pci_bridge0'.